src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -1830,13 +1831,29 @@ public HttpURLConnection makeConnection(String url, String method) throws IOExce
       connection.setRequestProperty(RequestConstants.TELEMETRY_INTEGRATION_VERSION, version);
     }
 
+    connection.setRequestProperty(HttpHeaders.USER_AGENT, buildUserAgentString().toString());
```

Comment:
Are you able to append to the header instead of setting it so that we can keep the other User-Agent information or is that automatic?

Reply:
Unfortunately, I don't think there's a way to get the default user-agent header that java builds so that we can prepend to it.

I tried this, but it returns `null`

```
connection.getRequestProperty("User-Agent");
```